src/components/MARCTable/MARCTableRow.test.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {
+  render,
+  cleanup,
+  fireEvent,
+} from '@testing-library/react';
+
+import '../../../test/jest/__mock__';
+import { Intl } from '../../../test/jest/__mock__/intl.mock';
+import { reduxFormMock } from '../../../test/jest/__mock__/reduxForm.mock';
+
+import { MARCTableRow } from './MARCTableRow';
+
+const onFieldUpdate = jest.fn();
+const removeSubfieldRows = jest.fn();
+const removeSubactionFromRow = jest.fn();
+const removeDataValuesFromRow = jest.fn();
+const fillEmptyFieldsWithValue = jest.fn();
+
+const mockedFunctions = [onFieldUpdate, removeSubfieldRows, removeSubactionFromRow,
+  removeDataValuesFromRow, fillEmptyFieldsWithValue];
+
+const renderMARCTableRow = () => {
+  const component = () => (
+    <Intl>
+      <MARCTableRow
+        subfieldIndex={0}
+        name="profile.mappingDetails.marcMappingDetails[0]"
+        order={0}
+        columnWidths={{}}
+        onAddSubfieldRow={() => {}}
+        onRemoveSubfieldRow={() => {}}
+        onFieldUpdate={onFieldUpdate}
+        removeSubfieldRows={removeSubfieldRows}
+        removeSubactionFromRow={removeSubactionFromRow}
+        removeDataValuesFromRow={removeDataValuesFromRow}
+        fillEmptyFieldsWithValue={fillEmptyFieldsWithValue}
+      />
+    </Intl>
+  );
+
+  return render(reduxFormMock(component));
```

Comment:
I think in the future it makes sense to create a custom render component which at least wraps children in the Intl components as down the road it should then be added to every test.

Reply:
I'm not sure I understand what you mean, could you clarify, please?

Reply:
I mean creating some sort of test setup component which wraps the content in intlProvider so in future it won't be duplicated over and over.

```
const renderWithIntl = { children } => {
   return render(<Intl intlProps={<some intl props if needed>}>{props.childrend})
}
```

Reply:
Done